### PR TITLE
Portfolio Setup: Added contact form and input validation

### PR DESCRIPTION
**Pull request by peterdtitan**
Connected the contact form with Formspree as required and updated the styling methods. The following files were changed:

- index.html: Finished the footer section
- style.css was updated to meet the given design template.

Screenshot 2022-06-01 at 9 37 01 AM

Screenshot 2022-06-01 at 9 38 07 AM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -185,5 +185,37 @@ <h2>
         </div>
     </section>
 
+    <section class="contact-me">
+        <div class="form">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="https://formspree.io/f/mlezzzeo" method="post" class="form">
+                <input id="name" name="name" type="text" placeholder="Full Name" required="required">
```
Comment (mikethreels) [inline index.html]: - [x] for the name field there should be a maximum of 30 characters allowed.

File: index.html [hunk 2]
```diff
@@ -185,5 +185,37 @@ <h2>
...
+                <input id="email" name="email" type="email" placeholder="Email Address" required="required">
+                <textarea name="message" rows="5" placeholder="Enter text here" required="required"></textarea>
```
Comment (mikethreels) [inline index.html]: - [x] for the message field there should be a maximum of 500 characters allowed.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Peter, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (peterdtitan) [reply index.html]: Fixed the maxlength attribute.

File: index.html (hunk 2 above)
Comment (peterdtitan) [reply index.html]: Fixed the maxlength attribute.

File: index.html [hunk 3]
```diff
@@ -185,5 +185,37 @@ <h2>
         </div>
     </section>
 
+    <section class="contact-me">
+        <div class="form">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="https://formspree.io/f/mlezzzeo" method="post" class="form">
+                <input id="name" name="name" type="text" placeholder="Full Name" required="required" maxlength="10">
```
Comment (mikethreels) [inline index.html]: - [x] the max length for the name input should be 30 according to the requirements.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Peter, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (peterdtitan) [reply index.html]: Slight typo error. Fixed.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @peterdtitan,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**